db/migrate/20170922175055_add_user_model.rb

```diff
@@ -1,8 +1,6 @@
 class AddUserModel < ActiveRecord::Migration[5.1]
   def change
     create_table :users do |t|
-      t.string :username, null: false
```

Comment:
I don't know where this field came from, but it is not in `schema.rb` and not used anywhere in our code.